package.json

```diff
@@ -32,18 +32,19 @@
     "package-tarball": "npm run build && npm pack"
   },
   "devDependencies": {
-    "@sketch-hq/sketch-assistant-utils": "6.0.0",
-    "@types/jest": "26.0.4",
-    "@types/node": "14.0.22",
-    "jest": "26.6.3",
+    "@sketch-hq/sketch-assistant-utils": "6.4.0",
+    "@types/jest": "26.0.23",
+    "@types/node": "15.12.0",
+    "jest": "27.0.4",
     "node-fetch": "2.6.1",
-    "prettier": "2.0.5",
-    "ts-jest": "26.1.2",
-    "typescript": "3.9.6",
-    "webpack": "4.43.0",
-    "webpack-cli": "3.3.12"
+    "prettier": "2.3.0",
+    "ts-jest": "27.0.2",
+    "typescript": "4.3.2",
+    "webpack": "5.38.1",
+    "webpack-cli": "4.7.0"
   },
   "dependencies": {
-    "@sketch-hq/sketch-assistant-types": "6.0.0"
+    "@sketch-hq/sketch-assistant-types": "6.4.0",
+    "@sketch-hq/sketch-file": "1.0.5"
```

Comment:
Can you make the change in [`sketch-assistants`](https://github.com/sketch-hq/sketch-assistants/blob/986cddcc8d9ae2ab357a9f41bd5593ee6a4d8e2a/packages/utils/package.json#L39) moving `@sketch-hq/sketch-file` to `dependencies` so we can drop it here?

Reply:
Sure

Reply:
https://github.com/sketch-hq/sketch-assistants/pull/185